epub33/core/index.html

```diff
@@ -5963,6 +5963,15 @@ <h4>URLs in the OCF Abstract Container</h4>
 						of <a data-cite="url#concept-url-parser">parsing</a> the file's <a>File Path</a> with the
 							<a>container root URL</a> as <a data-cite="url#concept-base-url"><var>base</var></a>.</p>
 
+					<div class="note">
+						The constraints on a <a>container root URL</a> are automatically honored for "usual" URLs like,
+						for example,  <code>https://localhost:12345/</code> or <code>https://www.example.org:12345/</code>. However,
+						a Reading System may chose a different strategy to identify the files of an EPUB container like, for example,
+						<code>file://path/to.epub#path=/EPUB/content.opf</code> or <code>jar:file:/path/to.epub!/EPUB/content.opf</code>;
+						in such a case the Reading System have to implement its own version of URL parsing to be conform to the conceptual
```

Comment:
A few more nitpicks:
- better to use a period than a semicolon to separate (it's a hard stop from the "however" clause)
- comma after "case"
- s/have/has/
- s/to be/to/

Reply:
Yep and yep and yep and yep.